ui/app/src/models/apiEndpoints.ts

```diff
@@ -20,5 +20,7 @@ export enum ApiEndpoint {
   Costs = "costs",
   Metadata = ".metadata",
   Health = "health",
-  Users = "users",
+  Users = 'users',
```

Comment:
These last three have been added with single quotes, which is inconsistent with the preceding items.
Suggest maintaining double quotes.